backend/infrahub/workflows/initialization.py

```diff
@@ -37,6 +38,9 @@ async def setup_deployments(client: PrefectClient) -> None:
         await workflow.save(client=client, work_pool=work_pool)
         log.info(f"Flow {workflow.name}, created successfully ... ")
 
+    if service:
+        await service.workflow.submit_workflow(workflow=SCHEMA_UPDATED_SETUP)
```

Comment:
I think we should just call the function from `setup_task_manager` after `setup_deployments` instead of starting a new workflow
it will take a bit more time but it's more inline with how the rest of the function is working right now and it avoids running into race condition if something running just after this function is expecting SCHEMA_UPDATED_SETUP to be completed

Reply:
Sounds good with me, instead of importing the function I continued with the name from the catalog and used the .get_function() from there.